valhalla/sif/edgelabel.h

```diff
@@ -416,7 +416,8 @@ class EdgeLabel {
   // be part of a bit field make sure kInvalidLabel is changed.
   uint32_t predecessor_;
 
-  // path_distance_: Accumulated path distance in meters.
+  // path_distance_: Accumulated path distance in meters. Can have a different meaning per algorithm,
+  //   e.g. represents walking distance in sif/multimodal.cc
```

Comment:
update this to have all distances, including transit, not only walking and re-introduce the member variable for walking distance